package/MDAnalysis/coordinates/XTC.py

```diff
@@ -64,10 +64,12 @@ def __init__(self, filename, n_atoms, convert_units=True,
             number of atoms to write
         convert_units : bool (optional)
             convert into MDAnalysis units
+        dt : float (optional)
+            timestep in MDAnalysis units to load trajectory with
         precision : float (optional)
             set precision of saved trjactory to this number of decimal places.
         """
-        super(XTCWriter, self).__init__(filename, n_atoms, convert_units,
+        super(XTCWriter, self).__init__(filename, n_atoms, convert_units, dt,
```

Comment:
use kwargs for the super call (sorry that convert_units was not done)
```suggestion
        super(XTCWriter, self).__init__(filename, n_atoms, convert_units=convert_units, dt=dt,
```
(you probably need to line-break ... actually, just run through black)